src/repo_extractor/schema.py

```diff
@@ -218,6 +218,8 @@ def _get_userlogin(api_obj) -> str:
     "output_path": _str_type,
     "api_domain_label_listing": _str_type,
     "api_subdomain_label_listing": _str_type,
+    "classification_method": _str_type,
+    "classification_model_save": _str_type,
```

Comment:
We should add documentation to support these additions to the schema, including the label listings. The extractor already has documentation for its original schema, so we just need to add these four items, and the names of the classification keys should be changed to match whatever changes we make to them in the source code after this review.